Update extended-support-release.rst (#4748)
mattermost/docs: source/upgrade/extended-support-release.rst

What is the life cycle of an Extended Support Release?
------------------------------------------------------

> Mattermost provides an ESR when the product gets an addition of a significant number of new features and improvements, and those new features have had sufficient time to stabilize. A new ESR is released twice a year, every January and July release. An ESR is supported for 9 months to provide customers with enough time to test and upgrade to the next ESR.

Reviewer: ```suggestion
Mattermost provides an ESR when a significant number of new features and improvements have been added to the product, and have had sufficient time to stabilize. A new ESR is released twice a year, every January and July release. An ESR is supported for 9 months to provide customers with enough time to test and upgrade to the next ESR.
```
